### PR TITLE
🚧 [WIP][NEW SIGNUP] Introduce a new Sign-in Flow Composer

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose

Add a WIP initial cut of the newly proposed Sign up flow builder.

Screenshot 2024-11-01 at 09 36 32

### Related Issues

- https://github.com/wso2/product-is/issues/21562

### Related PRs

- N/A

### Checklist

- [ ] e2e cypress tests locally verified. (for internal contributers)
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Relevant backend changes deployed and verified
- [ ] [Unit tests](/docs/testing/UNIT_TESTING.md) provided. (Add links if there are any)
- [ ] [Integration tests](https://github.com/wso2/product-is/tree/master/modules/integration) provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines
- [ ] Ran FindSecurityBugs plugin and verified report
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets

### CODE REVIEW

File: features/admin.authentication-flow-builder-core.v1/components/nodes/step-node.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```
Comment (dasuni-30) [inline features/admin.authentication-flow-builder-core.v1/components/nodes/step-node.scss:2]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

File: features/admin.authentication-flow-builder-core.v1/components/nodes/step-node.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2023-2024, WSO2 LLC. (https://www.wso2.com).
```
Comment (dasuni-30) [inline features/admin.authentication-flow-builder-core.v1/components/nodes/step-node.tsx:2]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Comment (brionmario) [reply features/admin.authentication-flow-builder-core.v1/components/nodes/step-node.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/7034/commits/37812d1f38beb7b8a1fc37e67c819e35068d9634

File: features/admin.authentication-flow-builder-core.v1/components/nodes/step-node.scss (hunk 1 above)
Comment (brionmario) [reply features/admin.authentication-flow-builder-core.v1/components/nodes/step-node.scss]: Fixed with https://github.com/wso2/identity-apps/pull/7034/commits/37812d1f38beb7b8a1fc37e67c819e35068d9634

**Pull request merged**